render/render.js

```diff
@@ -20,6 +20,7 @@ module.exports = function($window) {
 	}
 	function createNode(parent, vnode, hooks, ns, nextSibling) {
 		var tag = vnode.tag
+		vnode.reuse = true
```

Comment:
Re. underscore, no it is intended as a public field (to opt into recycling from hooks that would otherwise prevent a node from reaching the pool, when the user knows it's safe to do but we don't).

re. Vnode(), it's there, see https://github.com/lhorie/mithril.js/pull/1675/files#diff-44c4163c1ec81a946c6d826fe6d9c6dcR4